.github/workflows/welcome-first-time-contrib.yml

```diff
@@ -0,0 +1,13 @@
+name: Greeting the new contributor 
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Welcome to offical website repository for [json-schema.org](https://json-schema.org/). Thanks a lot for reporting your first issue. Please check out our [contributors guide](https://github.com/json-schema-org/website/blob/main/CONTRIBUTING.md) useful for opening a pull request.<br /> For more details check out [README.md](https://github.com/json-schema-org/website?tab=readme-ov-file#-welcome-to-the-json-schema-website) file.'
```

Comment:
> Please be sure you have reviewed the contributors guide for this and future PRs.

I think the way it's currently worded is a bit too late since they've already opened a PR.  It comes off better as a reminder in this case.